docs/DeveloperGuide.md

```diff
@@ -397,12 +397,17 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | user                                       | see the profile of a particular student           | get their information                                        |
 | `* * *`  | user                                       | remove a student's profile                        | keep track of only students that I teach                     |
 | `* * *`  | user                                       | edit the details of a student                     | keep track of up-to-date information                         |
-| `* * *`  | user                                       | add individual tuition sessions                   | keep track of my tuition sessions                            |
-| `* * *`  | user                                       | delete an individual tuition session              | update my tuition schedule                                   |
+| `* * *`  | busy user                                  | add an individual tuition sessions                | save time by creating multiple sessions in one command       |
+| `* * *`  | user                                       | add a recurring tuition session                   | keep track of my tuition sessions                            |
+| `* * *`  | user                                       | delete a particular tuition session               | update my tuition schedule                                   |
+| `* * *`  | user                                       | delete the entire recurring session               | update my tuition schedule                                   |
+| `* * *`  | user                                       | delete a single session from a recurring session  | take a break from lessons once in a while                    |
 | `* * *`  | user                                       | see a list of all the students profile and sessions|                                                             |
 | `* * *`  | user                                       | get all the emails of the parent of my students'  | email them reminders for payment                             |
 | `* * *`  | user                                       | calculate the monthly fees of a particular student| use the information when collecting monthly fees             |
 | `* * *`  | user                                       | get the monthly fees that I would have received for the past 3 months    | manage my finance better              |
+| `* * *`  | forgetful user                             | see a list of upcoming sessions                   | be aware of my teaching schedule for the next few days       |
+| `* * *`  | user teaching many lessons                 | see my schedule in a calendar view                | visual representation of my hectic schedules at a glance     |
```

Comment:
maybe the **so that I can** part can change to:
have an overview of my hectic schedules at a glance

Reply:
Ok! Sounds better! I have updated it in the latest commit! Thanks!